recipes/bob/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: bob
+  version: {{ version }}
+
+source:
+  fn: bob-{{ version }}.zip
+  url: https://pypi.io/packages/source/b/bob/bob-{{ version }}.zip
+  md5: b0119a365a0a79dda8bfdcba74f07e5f
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.ap ==2.0.4
+    - bob.blitz ==2.0.8
+    - bob.core ==2.1.2
+    - bob.db.atnt ==2.0.3
+    - bob.db.base ==2.0.6
+    - bob.db.iris ==2.0.4
+    - bob.db.mnist ==2.0.3
+    - bob.db.verification.utils ==2.0.7
+    - bob.db.wine ==2.0.3
+    - bob.extension ==2.0.11
+    - bob.io.audio ==2.0.0
+    - bob.io.base ==2.0.8
+    - bob.io.image ==2.0.6
+    - bob.io.matlab ==2.0.4
+    - bob.io.video ==2.0.6
+    - bob.ip.base ==2.0.9
+    - bob.ip.color ==2.0.4
+    - bob.ip.draw ==2.0.3
+    - bob.ip.facedetect ==2.0.7
+    - bob.ip.gabor ==2.0.5
+    - bob.ip.optflow.hornschunck ==2.0.6
+    - bob.ip.optflow.liu ==2.0.5
+    - bob.learn.activation ==2.0.4
+    - bob.learn.boosting ==2.0.7
+    - bob.learn.em ==2.0.8
+    - bob.learn.libsvm ==2.0.3
+    - bob.learn.linear ==2.0.7
+    - bob.learn.mlp ==2.0.11
+    - bob.math ==2.0.3
+    - bob.measure ==2.1.1
+    - bob.sp ==2.0.4
+
+  run:
+    - python
+    - bob.ap ==2.0.4
+    - bob.blitz ==2.0.8
+    - bob.core ==2.1.2
+    - bob.db.atnt ==2.0.3
+    - bob.db.base ==2.0.6
+    - bob.db.iris ==2.0.4
+    - bob.db.mnist ==2.0.3
+    - bob.db.verification.utils ==2.0.7
+    - bob.db.wine ==2.0.3
```

Comment:
all of these static pins make me very nervous about the future of this package.  Is there a reason you need to pin with ==?  Would >= be OK?

When you pin like this, the chances for unsatisfiable dependencies increase.